package/python-kiwi_boxed_plugin-spec-template

```diff
@@ -104,17 +129,35 @@ of images based on fast booting VM images
 %prep
 %setup -q -n kiwi_boxed_plugin-%{version}
 
-%build
-# Build Python 3 version
-%{__python3} setup.py build
+# Temporarily switch things back to docopt for everything but Fedora 41+
+# FIXME: Drop this hack as soon as we can...
+%if ! (0%{?fedora} >= 41 || 0%{?rhel} >= 10)
+sed -e 's/docopt-ng.*/docopt = ">=0.6.2"/' -i pyproject.toml
+%endif
+
+# Build documentation
+make -C doc man
+
+# Build application wheel
+%{__python3} -m build --no-isolation --wheel
 
 %install
-# Install Python 3 version
-%{__python3} setup.py install --prefix=%{_prefix} --root=%{buildroot} %{?is_deb:--install-layout=deb}
+# Install plugin
+%{__python3} -m installer --destdir %{buildroot} %{?is_deb:--no-compile-bytecode} dist/*.whl
+
+%if 0%{?is_deb}
+# Fix where files were installed
+mv %{buildroot}%{_prefix}/local/* %{buildroot}%{_prefix}
+mv %{buildroot}%{_prefix}/lib/python3* %{buildroot}%{_prefix}/lib/python3
+%endif
 
 # Install man pages and package documentation
 make buildroot=%{buildroot}/ docdir=%{_defaultdocdir}/ install
 
+%if 0%{?fedora} || 0%{?suse_version}
+%fdupes %{buildroot}/srv/tftpboot
+%endif
```

Comment:
Why is this here? We don't install any files in `/srv`.

Reply:
my bad, cut and paste bug :( Thanks for catching it and fixed